### ISSUE TITLE
upgrade to XState v5

### ISSUE THREAD

**Issue by mingfang** [tivac/xstate-component-tree#196]

**Comment by tivac** [comment]
Yup, I'd like to do it but I've been short on time and dayjob isn't able to move to v5 yet because it's tens of v4 machines and there's no automated way to port to v5 ([yet](https://github.com/statelyai/xstate-tools/pull/332#issuecomment-2365056406)).

**Comment by tivac** [comment]
https://github.com/tivac/xstate-component-tree/releases/tag/v7.0.1
